charter.html

```diff
@@ -174,10 +173,12 @@ <h2>Scope</h2>
             state.
         </p>
         <p>
-          The Working Group may consider designs that allow user agents for the
-          same user — including non-browser agents, like Operating Systems — to
-          collaborate in providing advertising features.
-          </p>
+          The Working Group will produce specifications that are implemented by
+          user agents.  The implementation of specifications in user agents
+          might depend on functionality provided by other entities.  In these
```

Comment:
```suggestion
          might depend on functionality provided by other entities or interoperate with non-web functionality.  In these
```

I agree that we functionally cannot include scope in the charter which is outside the remit of the W3C, however I do believe it is worth calling out that these specs may interop with things like operating systems (there are a number of examples of such APIs already providing some interop between "apps" and web, on both iOS and Android.)

Including this (along side the call out to functionality provided by other entities) makes it explicit that it can be a discussion point without adding it to the scope of the WG.

Reply:
I don't think "or interoperate with non-web functionality" adds much to the sentence. How about "might interoperate with or depend on functionality provided by other entities"?